### ISSUE TITLE
lightbox removes select and copy capabilty 

### ISSUE THREAD

**Issue by archie9211** [jhvanderschee/jekyllcodex#6]
after lightbox,one can select the text  only by double click.

this is probably due to preventDefault().

**Comment by jhvanderschee** [comment]
This is due to the swipe detection. Remove swipe detection and this 'bug' is gone. Yes... preventDefault probably causes this. Should I make the swipe detection optional? Would that help? Swipe detection is kind of a weird feature... Maybe it should be removed at all... What do you think?

**Comment by archie9211** [comment]
Yeah, I dont see usefullness of swipe detection. The main purpose of lightbox can be achieved without it.

**Comment by jhvanderschee** [comment]
In the new lightbox I have removed swipe. You can find it at https://jekyllcodex.org/blog/ditching-jquery-and-twitter-bootstrap/.

**Comment by DanilKorotenko** [comment]
Including lightbox in the head section fixed the issue.

Are there some specific reasons that lightbox should be included in the body bottom?

**Comment by DanilKorotenko** [comment]
I had the same issue.
When my site was hosted on GitHub pages, including lightbox in the head fixed this issue.
Then I moved my site to cloudcannon, and the issue appeared again.
I noticed that cloudcannon uses Jekyll 3.7.4 by default, while GitHub pages use Jekyll 3.8.5.
I added Gemfile to my site, and the text become selectable again. :)
I also, built my site locally to generate Gemfile.lock.
Gemfile content:
``
source 'https://rubygems.org'

gem "github-pages"
``
[Gemfile.zip](https://github.com/jhvanderschee/jekyllcodex/files/3819027/Gemfile.zip)
